sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/MemorySinkSuite.scala

```diff
@@ -70,35 +68,9 @@ class MemorySinkSuite extends StreamTest with BeforeAndAfter {
     checkAnswer(sink.allData, 1 to 9)
   }
 
-  test("directly add data in Append output mode with row limit") {
```

Comment:
Nit: I'd kinda prefer reverting as a separate PR

Reply:
I thought about it, but I didn't want to have the feature not working (even though no one is probably using it), and figured it would be easier this way since there's basically no file overlap between the two (except for a test class).